core/templates/pages/skill-editor-page/questions-tab/skill-questions-tab.directive.ts

```diff
@@ -75,9 +75,23 @@ angular.module('oppia').directive('questionsTab', [
           };
           ctrl.$onInit = function() {
             _init();
-            $scope.$on(EVENT_SKILL_INITIALIZED, _init);
-            $scope.$on(EVENT_SKILL_REINITIALIZED, _init);
+            ctrl.parentSubscription.add(
+              SkillEditorStateService.getSkillInitializedSubject()
+                .subscribe(
+                  () => _init()
+                )
+            );
+            ctrl.parentSubscription.add(
+              SkillEditorStateService.getSkillReinitializedSubject()
```

Comment:
Always break after `(` . Please follow the same pattern suggested above? Thanks!

Reply:
Done!